src/rtsp/stream_decoding.md

```diff
@@ -0,0 +1,126 @@
+# Actually using the stream
+What use is a stream if one cannot view it?
+
+When we were working with RTSP, we attempted to:
+1. deliver the stream to a WebRTC peer, and
+2. convert the received stream to an HLS playlist.
+
+Unfortunately, to decode the stream, we must first deal with several issues.
+
+## Keyframes
+Our implementation of a WebRTC Engine, the heart of Jellyfish, requires new streams to start with a keyframe (IDR frame).
```

Comment:
The idea of Jellybook is to provide theoretical concepts behind media server so we avoid references to RTC Engine, Jellyfish itself or videoroom. 

Instead of referencing RTC Engine and its alignment to the keyframe, we might write about a new webrtc peer joining to some room. In such a case we need to request keyframes from all other participants so the problem is the same but it doesn't sound like it's because of the engine architecture